src/panel/poi/blocks/Covid19.jsx

```diff
@@ -76,7 +76,7 @@ const LocalizedWarning = ({ countryCode }) => {
     'FR': (
       <div>
         <p>
-          {_('During the entire period of containment, movement to this location is only permitted with an overriding movement certificate.', 'covid19')}
+          {_('During the entire period of lockdown, movement to this location is only permitted with an overriding movement certificate.', 'covid19')}
```

Comment:
Apparently, instead of `overriding`, the good term is `derogatory`.
I'm not sure about `movement` either…

Reply:
Ok, I updated the string, can you review one more time please ?